docs/topics/topic-s2i-build-process.adoc

```diff
@@ -0,0 +1,15 @@
+= The Source-to-Image (S2I) Build Process
+
+link:https://docs.openshift.com/container-platform/3.4/architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image] is a build tool that allows you to generate reproducible Docker-formatted container images from application sources hosted in an online SCM repository.
+With S2I builds you can  easily deliver the latest version of your application into production while achieving shorter build times, decreased resource and network usage, improved security and a number of link:https://docs.openshift.com/container-platform/3.4/architecture/core_concepts/builds_and_image_streams.html#source-build[additional advantages].
+
+You need to provide three key elements as the input to the S2I process:
+
+* application sources (hosted in an online SCM repository, such as a Git Hub repo),
+
+* link:https://docs.openshift.com/container-platform/3.4/creating_images/s2i.html#s2i-scripts[S2I scripts],
+
+* the Builder image, which serves as the foundation for the assembled image and provides the ecosystem within which you application is running.
+
```

Comment:
In the above list, you need to either remove the punctuation at the end, or turn all 3 in to sentences and end them with periods